src/components/TextAreaChat/index.tsx

```diff
@@ -82,13 +82,14 @@ const TextAreaChat = ({
         multiline={true}
         onChangeText={onChangeText}
         placeholder={placeholder}
+        placeholderTextColor={COLORS.gray400}
         style={[
           S.textArea,
-          {color: disabledInput ? COLORS.lightgrey : COLORS.black, minHeight, maxHeight}
+          {color: disabledInput ? COLORS.lightgrey : COLORS.white2, minHeight, maxHeight}
         ]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.blackgrey` to `COLORS.gray400` and `COLORS.black` to `COLORS.white2` in the placeholder text and input text respectively might affect readability depending on the background color of the input field. Ensure that these changes maintain good contrast for accessibility.

```diff
-        placeholderTextColor={COLORS.blackgrey}
+        placeholderTextColor={COLORS.gray400}
```

```diff
-          {color: disabledInput ? COLORS.lightgrey : COLORS.black, minHeight, maxHeight}
+          {color: disabledInput ? COLORS.lightgrey : COLORS.white2, minHeight, maxHeight}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->